configuration-schema.json

```
@@ -209,6 +209,11 @@
               "type": "boolean",
               "description": "Strict defines whether the Overlay should be applied in a strict way, highlighting any actions that will not take any effect. This can, however, lead to more work when testing new actions in an Overlay, so can be turned off with this setting.",
               "default": true
+            },
+            "resolve-refs": {
+              "type": "boolean",
+              "description": "ResolveRefs enables reference resolving relative to the base path of the input file after the overlay is applied.",
+              "default": "false"
```

I appreciate you've done the work for wiring this in - I'd say this should be default behaviour

```suggestion
              "default": "true"
```

(I'd argue we could also not make it configurable at all, right now)